concept/type/attributetype.feature

```diff
@@ -143,24 +143,20 @@ Feature: Concept Attribute Type
     Then attribute(name) is abstract: true
     When transaction commits
     When session opens transaction of type: write
-    Then attribute(name) as(string) put: alice; throws exception
-    When session opens transaction of type: write
```

Comment:
remove cases that will throw for reasons other than what we want to test for here!